cmake/conan_utils.cmake

```diff
@@ -10,7 +10,7 @@ macro(setup_conan)
     # Right now every dependency shall be static
     set(CONAN_OPTIONS ${CONAN_OPTIONS} "*:shared=False")
 
-    set(REQUIREMENTS nlohmann_json/3.1.1 spdlog/1.9.2)
+    set(REQUIREMENTS nlohmann_json/3.1.1 spdlog/1.5.0)
```

Comment:
is there a reason of this downgrade?

Reply:
There is an inconsistency of version to https://github.com/doichanj/qiskit-aer/blob/24df5135cc06d9f9dd839d985b46bd91b2511232/cmake/dependency_utils.cmake#L27 and this causes build error on Power systems.